simulator/func_sim/trap_types.def

```diff
@@ -0,0 +1,9 @@
+TRAP(NO_TRAP             )
+TRAP(HALT                )
```

Comment:
My experience shows that aligning brackets is a bad practice. There is always a new item which requires to move everything.